esp-hal/src/spi/master.rs

```diff
@@ -1016,6 +1016,29 @@ pub mod dma {
         }
     }
 
+    impl<'d, T, C, M, DmaMode> crate::private::Sealed for SpiDma<'d, T, C, M, DmaMode>
+    where
+        T: InstanceDma<ChannelTx<'d, C>, ChannelRx<'d, C>>,
+        C: DmaChannel,
+        C::P: SpiPeripheral,
+        M: DuplexMode,
+        DmaMode: Mode,
+    {
+    }
+
+    impl<'d, T, C, M, DmaMode> InterruptConfigurable for SpiDma<'d, T, C, M, DmaMode>
+    where
+        T: InstanceDma<ChannelTx<'d, C>, ChannelRx<'d, C>>,
+        C: DmaChannel,
+        C::P: SpiPeripheral,
+        M: DuplexMode,
+        DmaMode: Mode,
+    {
+        fn set_interrupt_handler(&mut self, handler: crate::interrupt::InterruptHandler) {
+            SpiDma::set_interrupt_handler(self, handler);
+        }
+    }
```

Comment:
In theory yes - but the inherent function is already available always so I thought this is a bit artificial

It's also not much of a problem for async since IIRC the async implementation is using the DMA interrupt, not the SPI interrupt

Reply:
Ah fair enough